index.html

```diff
@@ -211,7 +211,57 @@ <h3 class="about-label">Skills</h3>
         <!-- End of About Section-->
         </section>
       <!-- End of Works Section -->
-     
+     <!-- Contact-Section -->
+      <section class="contact-section">
+        <h2 class="conatct-header">I'm always interested in hearing about new projects, 
+                                                            so if you'd like to chat  
+                                                            please get in touch.
+        </h2>
```

Comment:
- [ ] [OPTIONAL] If you change `hx` tag to `p` tag it would be more semantically!